packages/spotify_ml/spotify_ml.0.0.2/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Spotify API bindings"
+description: "Interact with the Spotify Web API via OCaml."
+maintainer: ["maxrn"]
```

Comment:
```suggestion
maintainer: ["maxrn"]
```
It would be useful to add a way to contact you here

Reply:
Thanks, updated.